feat: adds unit tests for EmployeeManager and Employee classes

bryanasb251 opened the pull request:
Assistance from Ezra and chat gpt.

anthonydmays reviewed (changes requested):
You also needed to implement the `getDetails` method in the `Employee` class in order to complete the assignment.

anthonydmays commented:
Closing for now, but feel free to reopen if you want to get more points by addressing the feedback.

bryanasb251 commented:
Ok I will address it next week so a lot is off the plate.

On Sat, Apr 12, 2025 at 2:53 PM Anthony D. Mays ***@***.***>
wrote:

> Closing for now, but feel free to reopen if you want to get more points by
> addressing the feedback.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/code-differently/code-differently-25-q1/pull/505#issuecomment-2798952506>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BQCIEBN3WRCPZCYSIG6Z7LT2ZFOJHAVCNFSM6AAAAAB26QSVQKVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDOOJYHE2TENJQGY>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
> *anthonydmays* left a comment
> (code-differently/code-differently-25-q1#505)
> <https://github.com/code-differently/code-differently-25-q1/pull/505#issuecomment-2798952506>
>
> Closing for now, but feel free to reopen if you want to get more points by
> addressing the feedback.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/code-differently/code-differently-25-q1/pull/505#issuecomment-2798952506>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BQCIEBN3WRCPZCYSIG6Z7LT2ZFOJHAVCNFSM6AAAAAB26QSVQKVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDOOJYHE2TENJQGY>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
